### PR TITLE
Test run bootstrap twice

### LINKED ISSUE #741: Check that running the bootstrap twice is working
Since the bootstrap should be idempotent, running it twice should work as usual.

Blocked by #717 since this double running should be run as a test and not as part of the default deployment process.

### PR DESCRIPTION
**Pull request by ramanduh**
Run a successive run of bootstrap, result we should not have error.

To test: Run the tests as described in `README.md`

Note: test can take time as the bootstrap is run twice

Closes #741

### CODE REVIEW

File: tests/post/steps/conftest.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    with open(version_file) as stream:
+        # add a section header for configparser
+        stream = chain(("[top]",), stream)
+        parser.readfp(stream)
+
+    return {k: v for k, v in parser.items('top')}
+
+
+@pytest.fixture
+def verify_kubeapi_service(host):
```
Comment (sayf-eddine-scality) [inline tests/post/steps/conftest.py]: we can put common steps in the conftest.py (that is why I created it in the first place)

File: tests/post/steps/test_bootstrap.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+from pytest_bdd import (
+    given,
+    scenario,
+    then,
+    when,
+)
+
+
+def _run_bootstrap(host, version):
```
Comment (sayf-eddine-scality) [inline tests/post/steps/test_bootstrap.py]: again, I think this should go to conftest.py

**Review by sayf-eddine-scality — CHANGES_REQUESTED** [review]

File: tests/post/steps/conftest.py (hunk 1 above)
Comment (ramanduh) [reply tests/post/steps/conftest.py]: In fact I is not really common steps as it is used by a `Given` steps in the first test and a `Then` step in the 2nd test. I did not find a way to alias the 2 steps, that's why I create the common fixture in the `conftest.py`

Comment (ramanduh) [reply tests/post/steps/conftest.py]: I moved it to the conftest

File: tests/post/steps/conftest.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+    Version is extracted from _build/root/product.txt
+    """
+    version_file = "_build/root/product.txt"
+    parser = ConfigParser()
+
+    with open(version_file) as stream:
+        # add a section header for configparser
+        stream = chain(("[top]",), stream)
+        parser.readfp(stream)
```
Comment (sayf-eddine-scality) [inline tests/post/steps/conftest.py]:
```
metalk8s/tests/post/steps/conftest.py:26: DeprecationWarning: This method will be removed in future versions.  Use 'parser.read_file()' instead.
    parser.readfp(stream)
```

File: tests/post/steps/conftest.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+    return {k: v for k, v in parser.items('top')}
```
Comment (sayf-eddine-scality) [inline tests/post/steps/conftest.py]: parser.items() is a dict ? why not returning it directly ?

**Review by sayf-eddine-scality — CHANGES_REQUESTED** [review]
ytest_bdd.exceptions.StepDefinitionNotFoundError: Step definition is not found: Given "the Kubernetes API is available". Line 8 in scenario "Exec in Pods" in the feature "metalk8s/tests/post/features/pods_alive.feature

File: tests/post/steps/conftest.py (hunk 3 above)
Comment (ramanduh) [reply tests/post/steps/conftest.py]: fixed

File: tests/post/steps/conftest.py (hunk 4 above)
Comment (ramanduh) [reply tests/post/steps/conftest.py]: It is a list. Something like that:
```
[('name', 'MetalK8s'), ('version', '2.0.0-dev'), ('short_version', '2.0'), ('git', '2.0.0-alpha1-7-gbb0bf9c'), ('development_release', '1'), ('build_timestamp', '2019-03-13T17:09:47Z'), ('build_host', 'duhamel-Latitude-E7450')]
```

File: tests/post/steps/test_bootstrap.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
+from pytest_bdd import (
+    given,
+    scenario,
+    then,
```
Comment (sayf-eddine-scality) [inline tests/post/steps/test_bootstrap.py]: not important but unused imports

**Review by sayf-eddine-scality — APPROVED** [review]

**Comment by ramanduh** [comment]
@bert-e approve

**Pull request merged**